isso/ext/notifications.py

```diff
@@ -168,24 +201,83 @@ def __init__(self, conf):
 
     def __iter__(self):
 
-        yield "comments.new:new-thread", self._new_thread
-        yield "comments.new:finish", self._new_comment
+        yield "comments.new:after-save", self._new_comment_after_save
         yield "comments.edit", self._edit_comment
+        yield "comments.new:new-thread", self._new_thread
+        yield "comments.new:finish", self._new_comment_finish
         yield "comments.delete", self._delete_comment
         yield "comments.activate", self._activate_comment
 
-    def _new_thread(self, thread):
-        logger.info("new thread %(id)s: %(title)s" % thread)
-
-    def _new_comment(self, thread, comment):
-        logger.info("comment created: %s", json.dumps(comment))
+    def _new_comment_after_save(self, thread, comment):
+        c = copy.copy(comment)
+        if 'voters' in c:
+            del c['voters']
+        logger.info("comments.new:after-save: %s, %s",
+                    json.dumps(thread), json.dumps(c))
 
     def _edit_comment(self, comment):
-        logger.info('comment %i edited: %s',
-                    comment["id"], json.dumps(comment))
+        c = copy.copy(comment)
+        if 'voters' in c:
+            del c['voters']
+        logger.info('comments,edit: %s, %s',
+                    json.dumps(thread), json.dumps(c))
+
+
+    def _new_thread(self, thread):
+        logger.info("comments.new:new-thread %(id)s: %(title)s" % thread)
+
+    def _new_comment_finish(self, thread, comment):
+        c = copy.copy(comment)
+        if 'voters' in c:
+            del c['voters']
+        logger.info("comments.new:finish: %s, %s",
+                    json.dumps(thread), json.dumps(c))
 
     def _delete_comment(self, id):
-        logger.info('comment %i deleted', id)
+        logger.info("comments.delete %i ", id)
 
     def _activate_comment(self, id):
-        logger.info("comment %s activated" % id)
+        logger.info("comments.activate %s" % id)
+
+
+import subprocess
+
+class Syscall(object):
+
+    def __init__(self, isso):
+        self.isso = isso
+        self.conf = isso.conf.section("syscall")
+        gh = isso.conf.get("general", "host")
```

Comment:
`gh` is unclear. as a name. Use something like `host`, `config_host`… whatever feels the best to you, but explicit.
Also please check that the recently added `public-endpoint` does not fit your needs here.